proto/ommx/v1/instance.proto

```diff
@@ -31,4 +31,13 @@ message Instance {
   //
   // TODO: Add semi-definite constraints to represent SDP
   // repeated SemiDefiniteConstraint semi_definite_constraints = ?;
+
+  // The sense of this instance
+  enum Sense {
+    SENSE_UNSPECIFIED = 0;
+    SENSE_MINIMIZE = 1;
+    SENSE_MAXIMIZE = 2;
+  }
+
+  Sense sense = 5;
```

Comment:
```suggestion
  // The sense of this problem, i.e. minimize the objective or maximize it.
  //
  // Design decision note:
  // - This is a required field. Most mathematical modeling tools allow for an empty sense and default to minimization. Alternatively, some tools do not create such a field and represent maximization problems by negating the objective function. This project prefers explicit descriptions over implicit ones to avoid such ambiguity and to make it unnecessary for developers to look up the reference for the treatment of omitted cases.
  //
  Sense sense = 5;
```

追加することには賛成ですが、ここは設計上の判断なのでNoteを残すべきです